tests/kernel/threads/no-multithreading/boards/bt510.conf

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
I don't see any changes to testcase.yaml to use this configuration or the overlay file. Need to wire this to CI.

Reply:
This test case will fail without this overlay - I2C drivers do not support single threaded operation.